src/http/ngx_http_wasm_api.c

```diff
@@ -37,6 +37,24 @@ typedef struct {
         return NGX_ERROR; \
     }
 
+enum {
+    WASM_H2_HEADER_PATH = 1,
+    WASM_H2_HEADER_METHOD,
+};
+
+typedef struct {
```

Comment:
Please group the definition according to the type. And use two blank lines as separator.

Reply:
updated